### ISSUE TITLE
ImportError: No module named indextank.client

### ISSUE THREAD

**Issue by haarts** [LinkedInAttic/indextank-service#3]
It probably is just me but running (step 1 of the 'getting started' paragraph):

```
python manage.py syncdb
```

results in

```
ImportError: No module named indextank.client
```

This is reference in `./nebu/api_linked_models.py` line 5.

**Comment by Kalyse** [comment]
I also got this. Did you manage to resolve it haarts?

**Comment by haarts** [comment]
Nope. I removed the offending line. The command then ran till some MySQL error, that error was wholly on my part, I didn't create a DB yet.

**Comment by dbuthay** [comment]
you can solve the problem by installing [https://github.com/flaptor/indextank-py].

If you have **pip** installed on your system, just run

`pip install indextank`

**Comment by hackaway** [comment]
I am getting this in a virtual env on fedora and i have the indextank installed.
i get this on running:  pip freeze

Django==1.2.7
MySQL-python==1.2.3
anyjson==0.3.1
indextank==1.0.7
uWSGI==1.2.3
wsgiref==0.1.2
Have there been any recent changes?

**Comment by hackaway** [comment]
To add to my previous comment: i get the error when running <path-to-virtual-env>/bin/python manage.py syncdb

**Comment by hackaway** [comment]
That is 
path-to-virtual-env/bin/python manage.py syncdb
